cypress/integration/plugins/reports-dashboards/01-create.spec.js

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { visitReportingLandingPage } from "../../../utils/plugins/reports-dashboards/constants";
+import { BASE_PATH } from '../../../utils/constants';
+
+describe('Adding sample data', () => {
+  it('Adds sample data', () => {
+    cy.visit(`${BASE_PATH}/app/home#/tutorial_directory/sampleData`);
+    cy.get('div[data-test-subj="sampleDataSetCardflights"]').contains(/(Add|View) data/).click();
+    cy.wait(3000);
```

Comment:
I'd recommend increasing this to 10s or something, seems it could get flaky loading all of that data in 3s